ecommerce/shopper_experience.rst

```diff
@@ -1,13 +1,8 @@
 ========
-Get paid
+Get Paid
```

Comment:
If your commit is about removing [REM] a doc, then this change
- shouldn't be part of this commit
- could be put in another commit

Also, I don't think it's that important. (To clarify things about capitalization, I'll talk with Julien and see if we put it in the guidelines or not)